cpp/src/io/parquet/parquet_gpu.h

```diff
@@ -133,7 +133,8 @@ struct ColumnChunkDesc {
                                      int8_t converted_type_,
                                      int8_t decimal_scale_,
                                      int32_t ts_clock_rate_,
-                                     int32_t col_index_)
+                                     int32_t col_index_,
```

Comment:
Since we're here, I would suggest making this name clearer.
```suggestion
                                     int32_t dst_col_index_,
```

Reply:
Done